Pod/Classes/CollectionViewHelpers.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CollectionViewHelpers.swift
+//  Pods
+//
+//  Created by Jason Grandelli on 12/6/16.
+//
+//
+
+import Foundation
+
+extension UICollectionView {
+    public final func registerReusableCell<T>(_ cell: ReusableCell<T>) where T: UICollectionViewCell {
```

Comment:
Yeah I don't see a good way around this for now. Maybe the `ReusableCell` could hold a reference to the class type as a property and then the generic wouldn't need to be enforced via the `register` function call. That's probably not worth the effort to change though.